src/Microsoft.Extensions.HealthChecks.Abstractions/Class1.cs

```diff
@@ -1,8 +0,0 @@
-﻿using System;
-
-namespace Microsoft.Extensions.HealthChecks.Abstractions
-{
-    public class Class1
```

Comment:
This really seems like an omission. I lot of customers I'm sure have code like this. If we ship it as an abstraction it will save them a lot of time

Reply:
Good point. Have we considered shipping a WebApplication1 package as well?

Reply:
Webmatrix actually did that. No joke. Like, literally not joking.

Reply:
Oh man, I do remember something like that happening.